src/main/java/core/basesyntax/service/impl/WriteServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WriteServiceImpl implements WriteService {
+    @Override
+    public void writeReport(File toFileName, String report) {
```

Comment:
```suggestion
    public void writeReport(String path, String report) {
```
lets pass string and convert it to Path inside method

Reply:
fixed